t/40-script_openqa-clone-custom-git-refspec.t

```diff
@@ -23,7 +23,6 @@ use Test::Warnings;
 use Test::Output;
 use OpenQA::Test::Utils qw(run_cmd test_cmd);
 
-
```

Comment:
unrelated change, please revert. If you want to apply whitespace changes please suggest them in a separate PR.

Reply:
To you seriously think to have PR for delete single new line?